tests/sentry/relay/test_config.py

```diff
@@ -179,6 +179,7 @@ def test_project_config_uses_filter_features(
     default_project.update_option("sentry:error_messages", error_messages)
     default_project.update_option("sentry:releases", releases)
     default_project.update_option("filters:react-hydration-errors", False)
+    default_project.update_option("filters:chunk-load-error", False)
```

Comment:
We should have a test to ensure this value produces a valid project config. If the feature is disabled, it won't be validated with the project config. If the config is invalid we will only see it in production when Relay is unable to parse the project config.

Reply:
Yep I was going to add it.

Reply:
Added the test in a follow up commit.